addon/components/o-s-s/search-field.stories.js

```diff
@@ -0,0 +1,62 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::SearchField',
+  component: 'search-field',
+  argTypes: {
+    value: {
+      description: 'Value of the input',
```

Comment:
```suggestion
    value: {
      type: { required: true },
      description: 'Value of the input',
```
According to the ts definition this attribute is mandatory